driver/csiplugin/controllerserver.go

```diff
@@ -961,23 +951,6 @@ func (cs *ScaleControllerServer) checkFileSetLink(ctx context.Context, connector
 	return nil
 }
 
-func checkFilesystemMountOnGUI(ctx context.Context, connector connectors.SpectrumScaleConnector, fsName string, scaleVolName string, primaryOrNone string) error {
-	loggerId := utils.GetLoggerId(ctx)
-	// primary filesytem must be mounted on GUI node so that we can create the softlink
-	// skip if primary and volume filesystem is same
-	klog.V(4).Infof("[%s] volume:[%v] - check if %s filesystem [%v] is mounted on GUI node of %s cluster", loggerId, scaleVolName, primaryOrNone, fsName, primaryOrNone)
-	isFsMounted, err := connector.IsFilesystemMountedOnGUINode(ctx, fsName)
```

Comment:
IsFilesystemMountedOnGUINode is also obsolete, right ?

Reply:
@badri-pathak , can you please check on this

Reply:
@saurabhwani5 @Jainbrt This is a part of gui rest call, written as a Interface, so will be keeping, might be used later. This won't come a lint issue.